Resources/en.lproj/Localizable.strings

```diff
@@ -1201,6 +1202,18 @@
 "toggle_public_transport" = "Show/hide public transport";
 "public_transport_show" = "Show public transport";
 "public_transport_hide" = "Hide public transport";
+"quick_action_showhide_mapillary_title" = "Show/hide Mapillary";
+"quick_action_showhide_mapillary_descr" = "A toggle to show or hide the Mapillary layer on the map.";
```

Comment:
Tapping this action button shows or hides the Mapillary layer on the map. (always use phrases that are similar to other actions)